introduccion-a-js-master/tareas/clase-5/task-4_styles.css

```diff
@@ -0,0 +1,34 @@
+.title {
+    margin: 40px 0 20px 40px;
+}
+
+.subtitle {
+    font-size: 1.15em;
+    margin: 10px 0 20px 40px;
+}
+
+.list-values-form {
+    margin: 50px 0 0 40px;
+}
+
+.values-form-group {
+    margin: 30px 0 10px 0;
+    border: none;
+}
+
+.value-field {
+    display: block;
+    margin-top: 10px;
+}
+
+.list-value {
+    margin-left: 5px;
+}
+.calculate-list-values-button {
+    display: none;
+}
+
+.results-of-the-calculations-of-the-list {
+    display: none;
+}
```

Comment:
Acá podes aglomerar los dos selectores así:
``.calculate-list-values-button,
.results-of-the-calculations-of-the-list  {
    display: none;
}
``